datalad/__init__.py

```diff
@@ -16,6 +16,11 @@
 from .config import ConfigManager
 cfg = ConfigManager()
 
+from datalad.support.sshconnector import SSHManager
```

Comment:
Okay. Misunderstood you, since the marked line was the import, not the construction.